qml/Components/PhysicalKeysMapper.qml

```diff
@@ -89,21 +98,31 @@ Item {
             if (event.isAutoRepeat && !d.ignoreVolumeEvents) root.volumeDownTriggered();
             else if (!event.isAutoRepeat) {
                 if (d.volumeUpKeyPressed) {
-                    if (Powerd.status === Powerd.On) {
+                    if (d.screenshotPressStart > 0
+                        && currentEventTimestamp - d.screenshotPressStart >= root.screenshotTimeout
```

Comment:
You'll need to move this out of the `(!event.isAutoRepeat)` condition somehow, or it will never be hit. See how the PowerDown/PowerOff keys above are handled.